src/crawlee/crawlers/_playwright/_playwright_crawler.py

```diff
@@ -210,6 +220,25 @@ async def _open_page(
                 await hook(pre_navigation_context)
         yield pre_navigation_context
 
+    def _prepare_request_handler(
```

Comment:
`request_handler` is already taken, let's come up with a better name. How does `_prepare_non_get_request_interceptor` sound? Not great, huh... but it's a start, I guess.